api/GPUAdapter.json

```
@@ -372,21 +372,39 @@
             "deprecated": false
           }
         },
-        "lost_device_on_duplicate": {
+        "duplicate_calls_reject": {
           "__compat": {
-            "description": "Lost `GPUDevice` returned on duplicate calls.",
+            "description": "Duplicate calls reject as adapter is consumed.",
             "tags": [
               "web-features:webgpu"
             ],
             "support": {
-              "chrome": {
-                "version_added": "116",
-                "partial_implementation": true,
-                "notes": "Supported on ChromeOS, macOS, and Windows only."
-              },
-              "chrome_android": {
-                "version_added": "121"
-              },
+              "chrome": [
+                {
+                  "version_added": "140",
+                  "partial_implementation": true,
+                  "notes": "Supported on ChromeOS, macOS, and Windows only."
+                },
+                {
+                  "version_added": "116",
+                  "version_removed": "140",
+                  "partial_implementation": true,
+                  "notes": [
+                    "Duplicate calls don't reject; instead, lost `GPUDevice`s are returned on duplicate calls.",
+                    "Supported on ChromeOS, macOS, and Windows only."
+                  ]
+                }
+              ],
```

Let's merge these (same for the browsers):

```suggestion
              "chrome": {
                "version_added": "116",
                "partial_implementation": true,
                "notes": [
                  "Supported on ChromeOS, macOS, and Windows only.",
                  "Before Chrome 140, lost `GPUDevice` is returned on duplicate calls."
                ]
              },
```

---

Fixed in latest commit.